docs/content/develop/resource.md

```diff
@@ -197,11 +197,11 @@ For more information about types of resources and the generation process overall
    - Sweepers: [`magic-modules/mmv1/third_party/terraform/utils`](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/utils)
 4. Modify the Go code as needed.
    - Remove the `Example` suffix from all test function names.
-   - Replace the comments at the top of the file with the following:
-     ```
-     <% autogen_exception -%>
-     ```
-   - If any of the added Go code (including any imports) is beta-only, change the file suffix to `.go.erb` and wrap the beta-only code in a version guard: `<% unless version == 'ga' -%>...<% else -%>...<% end -%>`.
+   - Remove the comments at the top of the file.
```

Comment:
In step 3, it mentions that the files are copied from the beta provider. I wonder if the files should be copied from the ga provider because of the import path. The import path for the packages (such as tpgresource, transport) should always be the path in the ga provider, even the packages are only needed in the beta version.

Reply:
Or we can tell the users to replace the package import path with the ga one.

Reply:
The reason for copying from beta is to make sure that beta-only code is included - nice catch regarding the import paths! +1 that this should be called out here. Fixed in 56f1a4f3a